packages/editor/src/store/actions.js

```diff
@@ -800,19 +800,19 @@ export function __experimentalDeleteReusableBlock( id ) {
 }
 
 /**
- * Returns an action object used in signalling that a reusable block's title is
+ * Returns an action object used in signalling that a reusable block is
  * to be updated.
  *
- * @param {number} id    The ID of the reusable block to update.
- * @param {string} title The new title.
+ * @param {number} id      The ID of the reusable block to update.
+ * @param {Object} changes The changes to apply.
  *
  * @return {Object} Action object.
  */
-export function __experimentalUpdateReusableBlockTitle( id, title ) {
+export function __experimentalUpdateReusableBlock( id, changes ) {
```

Comment:
I don't think reusable blocks functionality should be part of the editor store. Why can't we have and edit reusable blocks in the widgets screen? Maybe we should have a store for reusable blocks? I guess the work @epiqueras is doing with entities may also help us, maybe we can update the entity but don't save it right away and just update the entity of that reusable block.

Reply:
Fair question. I think to be honest we can probably remove all of this. Even without the refactor @epiqueras we have all what we need in the core store to fetch and save the reusable blocks (without edits since it's a separate flow). I just didn't want to make a bigger PR and tried to solve the main issue we had which is the fact that we were loading the parsed blocks of the reusable blocks in the default block editor reducer. (Hence the high number of fixed issues)

Reply:
We can use `EntityHandlers` to make a reusable block dead simple. I'm working on some updates that will make its flexibility clearer to you.

Reply:
I don't think `EntityHandlers` are of any help here because the consumer and the producer are at the same level here so we can just call the `core` module directly.

Reply:
It can do both. Like with the Site Title block: https://github.com/WordPress/gutenberg/pull/17020#issuecomment-521869286

You get a lot more stuff for free than if you use `core` directly.

Reply:
I know but I think it's wrong and useless. (we should probably continue this discussion in the other PR).

Reply:
>we have all what we need in the core store to fetch and save the reusable blocks

Yes, we can do it directly in `core/block`'s `edit`.